### PR TITLE
Display browsers as list

### PR DESCRIPTION
**Pull request by tomitm**
See #90.

image

TODO:
- [x] Display browsers as list
- [x] Look into identifying canary/dev separately
- [x] Fix issue with overflow if the window height is smaller than the list
- [x] browser-launcher uses browser name, instead of path?

### CODE REVIEW

File: src/component/main-content.js [hunk 1]
```diff
@@ -53,14 +53,34 @@ export default class MainContent extends React.Component {
...
+          if (a.version > b.version) return 1;
+
+          return 0;
+        })
+        .map((browser) => {
+          if (browser.name === 'ie') {
+            browser.name = 'Internet Explorer';
+          } else if (browser.type === 'chrome' && /SxS/.test(browser.command)) {
+            browser.name = 'Chrome Canary';
+          } else if (browser.type === 'firefox' && /Developer Edition/i.test(browser.command)) {
```
Comment (tomitm) [inline src/component/main-content.js]: Using `browser.command` might not work if the user changed their installation path, but the design still provides a fallback to the version number, so the user can differentiate between the two installations even if their names are the same.

**Comment by tomitm** [comment]
Two notes on this:
- Absolute positioning is giving me trouble with the layout right now (don't want to implement a height workaround with JS on this..)
- browser-launcher2 uses the browser name as a launch option? That's.. odd. Need to look into that.

**Comment by tomitm** [comment]
Fixed the overflow issue!

Although I noticed that `browser-launcher` uses the *browser name* to launch the browser...

**Comment by tomitm** [comment]
Looks like I can pass the name and version, so that's set! :tada: 

@mitchhentges or @nerdbeere can you have a look and merge?

**Comment by mitchhentges** [comment]
:+1:

**Pull request merged**